cmake/CMakeLists.txt

```diff
@@ -44,18 +63,17 @@ if (CMAKE_USE_PTHREADS_INIT)
   add_definitions(-DHAVE_PTHREAD)
 endif (CMAKE_USE_PTHREADS_INIT)
 
-if (MSVC)
-  if (ZLIB)
-    set(HAVE_ZLIB 1)
-    find_path(ZLIB_INCLUDE_DIRECTORIES zlib.h ${protobuf_SOURCE_DIR})
-    find_library(ZLIB_LIBRARIES zdll ${protobuf_SOURCE_DIR})
-  else (ZLIB)
-    set(HAVE_ZLIB 0)
-  endif (ZLIB)
-else (MSVC)
+if (ZLIB)
   find_package(ZLIB)
```

Comment:
I don't know how this simplified version is going to work with MSVC build with ZLIB. Could you update the instructions in cmake/README.md about the zlib build part in this pull request?